website/src/components/expandable/styles.module.css

```diff
@@ -59,6 +59,10 @@
     margin-right: 8px;
 }
 
+details[open] :local(.toggle)::before {
+  transform: rotate(225deg); /* this rotates the chevron down */
```

Comment:
hey @john-rock , added this to fix the chevron detail. can you please re-review when you have a chance?